data/posts/split-route-modules.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Split Route Modules
+summary: React Router v7.2's framework mode now supports automatic code splitting of route modules
+date: 2025-02-27
+image: /blog-images/headers/react-router-v7.jpg
+ogImage: /blog-images/headers/react-router-v7.jpg
+imageAlt: "The React Router logo"
+imageDisableOverlay: true
+authors:
+  - Mark Dalgleish
+---
+
+With the release of [React Router v7.2.0](https://github.com/remix-run/react-router/releases/tag/react-router%407.2.0), we’ve introduced a new opt-in framework feature called Split Route Modules. In this post, we’ll explore the performance problem that Split Route Modules solves, how it works, and how to use it today.
+
+Please note that this feature is currently experimental, enabled by the `future.unstable_splitRouteModules` flag. If you want to use it in production, we recommend testing against your production build to ensure that the optimization is working as expected.
```

Comment:
Slight rewording suggestion to better align with the wording in our docs (and add a cross-link):

```suggestion
Please note that this feature is currently [unstable](https://reactrouter.com/community/api-development-strategy#unstable-flags), enabled by the `future.unstable_splitRouteModules` flag. We'd love any interested users to play with it locally and provide feedback, but we do not recommend using it in production yet. If you do choose to adopt this flag in production, please ensure you do sufficient testing against your production build to ensure that the optimization is working as expected.
```

Reply:
Nice, I like this.